ci/test-05-options-c-e.pl

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/perl -w
 
-use Test::Command tests => 33;
+use Test::Command tests => 42;
```

Comment:
Please consider adding two error test cases for the new option:

1. omitting the format name, and
2. using an unsupported format name.

Reply:
I have add two negativ test `--timestamp-format` and `--timestamp-format="%Y-%m-%d %H:%M:%S"` both ending in function usage() with exit code 1